webapp/src/client/client.js

```diff
@@ -251,18 +242,14 @@ export default class Client {
             EndpointId: endpointId,
             Culture: 'en-US',
         };
-        const response = await request.
-            post(oauthApplicationHref).
-            set('Authorization', authorizationValue).
-            set('Accept', 'application/json').
-            send(data);
+        const response = await this.doPost(oauthApplicationHref, data, {Authorization: authorizationValue, Accept: 'application/json'}, 'omit');
```

Comment:
I don't think I understand the `omit` thing. Is this the CORS issue you were talking about?

Reply:
If we don't use omit here, we will have a CORS error when talking with the discovery service. Using 'omit' here silence that error.